api_contracts/contracts/downloads/CustomAccountDownload.md

```diff
@@ -32,6 +32,29 @@ This endpoint returns the generated file's metadata.
     + Attributes
         + results (array[CustomDataResult], fixed-type)
 
+## Budget Function [/api/v2/budget_functions/list_budget_functions/]
+
+This endpoint returns a list of budget functions with their associated title and code
+
+### Custom Budget Function [GET]
```

Comment:
We can call this just "Budget Function" since it could be used for other purposes by API users.